iquip/apps/explorer.py

```diff
@@ -96,6 +97,48 @@ def run(self):
         self.fetched.emit(experimentList, self.widget)
 
 
+class BuilderInfoDialog(QDialog):
+    """Dialog for setting builder info.
+    
+    There are two types of info.
+        clsName: Target experiment class name.
+        tag: Additive tag for multiple builder apps.
+    """
+
+    def __init__(self, clsNames: Iterable[str], parent: Optional[QWidget] = None):
+        """Extended.
+        
+        Args:
+            See thread.ExperimentInfoThread.fetched signal.
```

Comment:
Now this is actually not a proper redirection, but I think it is pretty intuitive enough. If you agree with it, LGTM. You can update the docstring as well.

Reply:
Then, let's keep it as it is.